.github/workflows/verify-app.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Capture Growth Chart: verify app'
+
+on:
+  push:
+    branches:
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+  lint:
```

Comment:
Add the name key for readability. Add it for all of the three jobs:

- `name: Install Dependencies`
- `name: Lint Code`
- `name: TypeScript Check`